Use fixed generic RGB color space when parsing color hex values

mark0815 opened the pull request.

mark-sprylab committed:
Use fixed generic RGB color space when parsing color hex values

clausweymann commented:
Looks fine to me. But just out of curiosity, what is the effect of this change? I want screenshots ;)

mark-sprylab committed:
Use sRGB color space when converting from/to HEX color representation

As we use a CSS notation for the colors we should use the color space described in the CSS Spec: sRGB. (http://www.w3.org/TR/css3-color/)
The actual color space is not that important, more important is that we use the same color space for encoding and decoding.

mark-sprylab committed:
Cleanup

mark-sprylab committed:
Modern syntax, Cleanup
